pkg/injector/sidecar/consts.go

```diff
@@ -27,6 +27,8 @@ const (
 	SidecarDebugPortName           = "dapr-debug"
 	SidecarHealthzPath             = "healthz"
 	SidecarInjectedLabel           = "dapr.io/sidecar-injected"
+	SidecarAppIdLabel              = "dapr.io/app-id"
```

Comment:
Please fix CI

Reply:
Oh yes, meant to do that last night.
Shouldn't `make check` pick this up?

I see mention of a specific version of the linter needed in the contributor docs, maybe `make check` needs fixing to pick this up when running locally? Or perhaps I just need to use the devcontainer :)

Reply:
Fixed

Reply:
`make check` should have checked this lint error, and please make sure you are using golangci-lint version v1.51.2.